Source/Core/DolphinQt/GCMemcardManager.h

```diff
@@ -22,6 +24,17 @@ class QPushButton;
 class QTableWidget;
 class QTimer;
 
+class GCMemcardManagerIconAnimationData
```

Comment:
If you don't intend to implement methods on this data:
```suggestion
struct GCMemcardManagerIconAnimationData
```
And then get rid of the `public:` part.